uint8 H_MIN
uint8 H_MAX
uint8 S_MIN
uint8 S_MAX
uint8 V_MIN
uint8 V_MAX